auth/resource.go

```diff
@@ -76,7 +77,9 @@ func (m *AuthzResourceManager) CreateSpace(ctx context.Context, request *http.Re
 			"response_status": res.Status,
 			"response_body":   rest.ReadBody(res.Body),
 		}, "unable to create a space resource via auth service")
-		return nil, errs.Errorf("unable to create a space resource via auth service. Response status: %s. Response body: %s", res.Status, rest.ReadBody(res.Body))
+		// Proxy-back back the response as is -
+		// WIT acts as a gateway to Auth, who would send the appropriate response.
+		return nil, proxy.ConvertHTTPErrorCode(ctx, res.StatusCode, rest.ReadBody(res.Body))
```

Comment:
res.Body is already empty here (was read when logging the error above). Read the body to a string then reuse it when logging the error and when converting the response to the http error.

Reply:
https://github.com/fabric8-services/fabric8-wit/pull/2026/commits/bf3efeef4ef3b66b35f1d3512daa55f24f2f8978